Exercises/LinusLausberg/exercises_08.py

```diff
@@ -0,0 +1,106 @@
+import os
+import numpy as np
+import textwrap as tw
+np.set_printoptions(threshold=np.inf)
+
+# --- Day 15: Chiton ---
+# You've almost reached the exit of the cave, but the walls are getting closer together. Your submarine can barely still fit, though; the main problem is that the walls of the cave are covered in chitons, and it would be best not to bump any of them.
+
+# The cavern is large, but has a very low ceiling, restricting your motion to two dimensions. The shape of the cavern resembles a square; a quick scan of chiton density produces a map of risk level throughout the cave (your puzzle input). For example:
+
+# 1163751742
+# 1381373672
+# 2136511328
+# 3694931569
+# 7463417111
+# 1319128137
+# 1359912421
+# 3125421639
+# 1293138521
+# 2311944581
+# You start in the top left position, your destination is the bottom right position, and you cannot move diagonally. The number at each position is its risk level; to determine the total risk of an entire path, add up the risk levels of each position you enter (that is, don't count the risk level of your starting position unless you enter it; leaving it adds no risk to your total).
+
+# Your goal is to find a path with the lowest total risk. In this example, a path with the lowest total risk is highlighted here:
+
+# 1163751742
+# 1381373672
+# 2136511328
+# 3694931569
+# 7463417111
+# 1319128137
+# 1359912421
+# 3125421639
+# 1293138521
+# 2311944581
+# The total risk of this path is 40 (the starting position is never entered, so its risk is not counted).
+
+# What is the lowest total risk of any path from the top left to the bottom right?
+
+def reading_relativ_cost():
+    lines = []
+    script_dir = os.path.dirname(os.path.abspath(__file__))
+    sequence_path = os.path.join(script_dir, "..", "..", "data",
+                                 "exercise_cave.txt")
+    with open(sequence_path, 'r') as inputfile:
+        lines = inputfile.readlines()
```

Comment:
Falls die Datei nicht gefunden wird, könntest du diese Fehlerbehebung hinzufügen:
```suggestion
try:
    with open(sequence_path, 'r') as inputfile:
        lines = inputfile.readlines()
except FileNotFoundError:
    print(f"Error: File not found at {sequence_path}")
    return None
```